requirements.txt

```diff
@@ -0,0 +1,10 @@
+mkdocs
```

Comment:
I suppose I should admit that despite being familiar with Python, I've never used `requirements.txt` before. In fact, when you mentioned it in Discord I thought maybe you were referring to some documentation file, not a functional list of packages.

I think as long as the `docs/requirements.txt` captures all the modules needed to run `mkdocs serve`, I'm happy to get rid of the one I added in favor of the existing one.

> * so that makes reproducibility impossible

Agreed. However I do not understand why reproducibility is important for a docs repo. Can you elaborate? The cost of putting version numbers here is an additional maintenance burden to upgrade packages manually. Not a huge burden, but one that requires someone's time. So I'd like to understand the justification for it.

Reply:
> > * so that makes reproducibility impossible
> 
> Agreed. However I do not understand why reproducibility is important for a docs repo. Can you elaborate? The cost of putting version numbers here is an additional maintenance burden to upgrade packages manually. Not a huge burden, but one that requires someone's time. So I'd like to understand the justification for it.

If you build the docs locally with version XYZ of a dependency and the docs are deployed with version ABC of that same dependency things could render differently due to changes in the dependency.

Maintaining and updating the dependency versions is handled automatically via pull requests from dependabot https://github.com/TRaSH-Guides/Guides/pulls?q=is%3Apr+author%3Aapp%2Fdependabot+ so there is no need for anyone to search for when updates are available, just for someone to review the pull request for any breaking change notes and potentially test building the docs locally before merging the dependency changes.

Reply:
Makes sense. Thank you for explaining. I will push a change to use the existing requirements.txt.